mods/dungeon_loot/loot.lua

```diff
@@ -0,0 +1,62 @@
+dungeon_loot.registered_loot = {
+	-- buckets
+	{name="bucket:bucket_empty", chance=0.55},
```

Comment:
Please can we have spaces around binary operators throughout the commit? It really helps readability.

Reply:
will do except for vectors, writing `{x = 0, y = 0, z = 1}` wastes a lot of space IMO

Reply:
Readability is more important, space is good. But ok.